modules/api/src/main/java/org/apache/fluo/api/client/FluoAdmin.java

```diff
@@ -119,6 +119,15 @@ public TableExistsException() {
   void initialize(InitializationOptions opts)
       throws AlreadyInitializedException, TableExistsException;
 
+  /**
+   * Removes Fluo application, Accumulo table and shared configuration in Zookeeper. Shared configuration
+   * consists of all properties except those with
+   * {@value org.apache.fluo.api.config.FluoConfiguration#CONNECTION_PREFIX} prefix.
+   * 
+   * @since 1.2.0
+   */
+  void remove();
```

Comment:
It would be really nice to have an integration test for this new API call.  The new test could do the following.  
 * Add some data.  Remove the app.  Create an app with same name. Verify no data.  Add some data.  Verify data.
 * Try to remove a running app.

This test may be tricky to write because a lot of the current test code assumes a single app.  I would be happy to write the test or provide guidance on writing the test, just let me know.

Reply:
I tested the remove command on my machine and everything worked.  An integration test is a great idea but should probably be a separate issue.

Reply:
I've notice we don't have many references (tests) on command module. If you have any reference a can learn how to do. But your guidance would be nice too. Thanks!

Reply:
> An integration test is a great idea but should probably be a separate issue.

Creating the test in a follow on PR is ok with me.

> I've notice we don't have many references (tests) on command module.

That's right, we do not have any automated test for the commands other than things like [stresso](https://github.com/astralway/stresso) that use them in scripts, but that does not cover all commands and options.   However the `remove()` call is in the Java API making it a bit easier to test in an IT.

Reply:
A good place to add a test for remove would be the following

https://github.com/apache/fluo/blob/dbad51d8a0763dc6ee21cee30d446c40e94c9ed8/modules/integration/src/test/java/org/apache/fluo/integration/client/FluoAdminImplIT.java

You can see examples of initialization in exiting test

https://github.com/apache/fluo/blob/dbad51d8a0763dc6ee21cee30d446c40e94c9ed8/modules/integration/src/test/java/org/apache/fluo/integration/client/FluoAdminImplIT.java#L62

This test has an example of stoping the oracle.  Remove should fail before stopping the oracle and succeed after.

https://github.com/apache/fluo/blob/dbad51d8a0763dc6ee21cee30d446c40e94c9ed8/modules/integration/src/test/java/org/apache/fluo/integration/client/FluoAdminImplIT.java#L155

Below is outline of what I think a test may look like.  Not sure if it compiles.  Also not sure if oserver.start() will have problems are reinit.

```java
 @Test
  public void testRemove() throws Exception {
    try (FluoAdmin admin = new FluoAdminImpl(config)) {
        admin.remove();
       Assert.assertFail("expected remove to fail");
    }catch(FluoException e) {
   }

   //TODO write some data

   oserver.stop();

    try (FluoAdmin admin = new FluoAdminImpl(config)) {
        admin.remove();
    }

   try (FluoAdmin admin = new FluoAdminImpl(config)) {
       //should succeed w/o clearing
       InitializationOptions opts =
          new InitializationOptions().setClearZookeeper(false).setClearTable(false);
       admin.initialize(opts);
    }

   //not sure if this will work properly
   oserver.start();

   //TODO verify empty
   //TODO write data
   //TODO read data

 }
```